tlg-base-config/azuredeploy.json

```diff
@@ -0,0 +1,568 @@
+            }
+        },
+        "localAdminPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'localAdmin' account on each of the Virtual Machines" 
+            }
+        },
+        "safemodeAdmin": {
+            "type": "string",
+            "defaultValue": "safemodeAdmin",
+            "metadata": { 
+	            "description": "The safemodeAdmin account for the Domain" 
+            }
+        },
+        "safemodeAdminPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'safemodeAdmin'" 
+            }
+        },
+        "genericUser": {
+            "type": "string",
+            "defaultValue": "User1",
+            "metadata": { 
+	            "description": "The account to be created and promoted to EA/DA" 
+            }
+        },
+        "genericUserPassword": {
+            "type": "securestring",
+            "metadata": { 
+	            "description": "Password for the 'genericUser' account" 
+            }
+        },
+        "newStorageAccountName": { 
+	        "type": "string", 
+	        "metadata": { 
+	            "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed. (Length: 3-24; Case: Lower; Valid: Alphanumeric; Unique: Global)" 
+            } 
+        }
+    },
+    "variables": {
+        "githubroot": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/tlg-base-config/dsc/",
+        "storageAccountType": "Standard_LRS",
+        "imageSku": "2012-R2-Datacenter",
+        "vmSize": "Standard_A1",
+        "virtualNetworkName": "TestLab",
+        "virtualSubnetName": "Corpnet",
+        "domainName": "corp\\",
+        "dscConfigZIP": "[concat(variables('githubroot'), 'dsc_config.zip')]",
+        "dscConfigFile": "dsc_config.ps1",
+        "dscConfigScript": "WindowsTestConfigEnv"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "APP1",
+            "apiVersion": "2015-06-15",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "hardwareProfile": {
+                    "vmSize": "[variables('vmSize')]"
+                },
+                "storageProfile": {
+                    "imageReference": {
+                        "publisher": "MicrosoftWindowsServer",
+                        "offer": "WindowsServer",
+                        "sku": "[variables('imageSku')]",
+                        "version": "latest"
+                    },
+                    "osDisk": {
+                        "name": "APP1-OSDisk",
+                        "createOption": "FromImage",
+                        "vhd": {
+                            "uri": "[concat('https://', parameters('newStorageAccountName'), '.blob.core.windows.net/vhds/APP1-OSDisk.vhd')]"
```

Comment:
see #9 here for constructing storage uris...

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#best-practices